server/bleep/src/lib.rs

```diff
@@ -338,25 +344,32 @@ impl FromRef<Application> for axum_extra::extract::cookie::Key {
     }
 }
 
-#[cfg(all(tokio_unstable, feature = "debug"))]
-fn tracing_subscribe() -> bool {
-    use tracing_subscriber::{fmt, prelude::*};
-    let env_filter = fmt::layer().with_filter(EnvFilter::from_env(LOG_ENV_VAR));
-    tracing_subscriber::registry()
-        .with(env_filter)
-        .with(sentry_layer())
-        .with(console_subscriber::spawn())
-        .try_init()
-        .is_ok()
-}
+fn tracing_subscribe(config: &Configuration) -> bool {
+    let env_filter_layer = fmt::layer().with_filter(EnvFilter::from_env(LOG_ENV_VAR));
+    let sentry_layer = sentry_layer();
+    let log_writer_layer = (!config.disable_log_write).then(|| {
+        let log_dir = config.index_dir.join("logs");
+        let file_appender = tracing_appender::rolling::hourly(log_dir, "bloop.log");
+        let (non_blocking, guard) = tracing_appender::non_blocking(file_appender);
+        _ = LOGGER_GUARD.set(guard);
+        fmt::layer()
+            .with_writer(non_blocking)
+            .with_ansi(false)
+            .with_filter(Targets::new().with_target("bleep", LevelFilter::DEBUG))
+            .with_filter(Targets::new().with_target("bleep::indexes::file", LevelFilter::WARN))
```

Comment:
`bleep::indexes::file` is the noisiest module of them all. we can limit this to `warn` and above, but have debug logs for the rest of the codebase (should give us insight into `webserver` just fine.)

Reply:
Can you recalculate the log size per hour with this change? After how many days should we cleanup logs?

Reply:
~130kB/hr; size shouldn't be a problem here, we can afford to keep logs for upto a month or more.